canonical/jaas-dashboard
# Display errors when trying to log into a controller.

## Done

- Display connection errors when logging in.
- Migrate Login component to TypeScript/RTL.

## QA

- Go to the login screen and type some incorrect credentials and submit.
- You should see an error.
- Change your config to point to an incorrect controller address.
- Go to the login screen and type some credentials and submit.
- You should see an error.

## Details

Fixes: #867.

## Screenshots

<img width="408" alt="Screenshot 2022-10-26 at 10 41 37 am" src="https://user-images.githubusercontent.com/361637/197902495-47a33f8d-dceb-436c-bfa5-2a7ab9b4d905.png">
